spec/factories/organisations_factory.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :organisation do
+    sequence(:id) { |index| index }
```

Comment:
I have moved it to this commit 309d54644f4a1b9ce1e29b0ed0b62f0759a47c13 where it is used